styles/style.css

```diff
@@ -0,0 +1,185 @@
+
+:root{
+    --clr-background: rgb(208, 225, 253);
+    --clr-prinmary:  rgb(99, 144, 228);
+    --clr-gray: #263238;
+    --clr-white: white;
+    
+    --font-size: 16px;
+    --border-radius: 0.2em;
+    --main-radius:5px;
+    --main-padding:5px;
+}
+
+
+
+#Bio {
+    text-align: center;
+    font-size: 16px;
+    font-family: "Open sans", sans-serif;
+    color: black;
+    
+}
+
+/* .bio-p{
+    color: black;
+    text-align: center;
+    margin: 2em 10em 2em 10em;
+    border-style: solid;
+    border-color: rgb(68, 102, 164);
+    font-size: 20px;
+    font-family: fantasy;
+
+} */
+
+
+a:link{
+    color: black
+}
+body{
+    margin: 0;
+    font-family: "Open Sans", sans-serif;
+    font-size: var(--font-size);
+    color: var(--clr-gray);
+    background-color: var(--clr-background);
+}
+
+.about-header{
+    padding: 0 0.2em;
+    font-size: 1.5em;
+    position: fixed;
+    left: 150px;
+}
+
+
+/* a{
+    font-family: "Open Sans", sans-serif;
+    text-decoration: none;
+    font-size: 1em;
+    color: black;
+
+} */
+
+
+.nav-list {
+    background: var(--clr-prinmary);
+    margin: 0;
+    padding: 1rem 0;
+    border-radius: var(--radius);
+    list-style: none;
+    box-shadow: 10px 10px 10px var(--clr-gray200);
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+}
+.nav-list a{
+    color: white;
+    text-decoration: none;
+    font-size: 1em;
+}
+
+li {
+    display: flex;
```

Comment:
I don't see any places where setting `flex` on an `li` has an effect. We only need to set `flex` if an element has children that we need to have laid out according to flex rules.